corehq/apps/app_manager/views/modules.py

```diff
@@ -1337,13 +1338,22 @@ def _check_xpath(xpath, location):
                 additional_registry_cases=additional_registry_cases,
                 custom_related_case_property=search_properties.get('custom_related_case_property', ""),
                 inline_search=search_properties.get('inline_search', False),
+                pull_parent_child_ext_cases=search_properties.get('pull_parent_child_ext_cases', False)
             )
+            _update_pull_parent_child_ext_cases(app, module,
+                search_properties.get('pull_parent_child_ext_cases', False))
 
     resp = {}
     app.save(resp)
     return JsonResponse(resp)
 
 
+def _update_pull_parent_child_ext_cases(app, current_module, new_state: bool):
+    for module in app.get_modules():
+        if module.case_type == current_module.case_type:
+            module.search_config.pull_parent_child_ext_cases = new_state
```

Comment:
I think I see why but see my comments below where I propose a different solution (putting this setting in the suite XML so that it get's sent with the case search request).